packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -22,13 +23,17 @@ class RemoteConfigManager extends EventEmitter {
   constructor (config) {
     super()
 
-    const pollInterval = config.remoteConfig.pollInterval * 1000
+    const pollInterval = Math.floor(config.remoteConfig.pollInterval * 1000)
+    const url = config.url || new URL(format({
+      protocol: 'http:',
+      hostname: config.hostname || 'localhost',
+      port: config.port
+    }))
```

Comment:
that should be done in config.js or somewhere else, otherwise we have to copy this code everywhere

Reply:
I agree, but right now this is done all over the codebase, so this is a larger effort our of scope of this PR.